docs/practices/workflows/benchmarking_synthetic_workloads.md

```diff
@@ -38,6 +38,18 @@ Before an RPC request is sent, the tooling awaits capacity on a buffered channel
 
 The tooling's [`justfile`](../../../benchmarks/synth-bm/justfile) contains recipes for the most relevant workflows.
 
+A typical workflow benchmarking the native token transfers using the above `justfile` would be something along the:
+```command
+[t1]$ just init_localnet
+[t1]$ just create_subaccounts
+# apply the desired config modifications (see below)
+[t2]$ just run_localnet
+# set the desired tx rate and total volume in the justfile (see above)
+[t1]$ just benchmark_native_transfers
+```
+Watch out! Running the `neard` via `just run_localnet` is going to pick whatever `release` executable currently compiled.
+So comparing different binaries one needs to be careful to (re-)compile the `neard` as needed.
```

Comment:
Of course the workaround is placing your binaries somewhere else and point the justfile to them :)

Though I see that this can be confusing, so I'll change the default behavior in a separate PR. So I think you can remove these two sentences.

Reply:
I'm for going with #12817 and removing these two sentences here, because I had some issues too with the justfile picking up `/target/release/neard`.